src/formatDate.js

```diff
@@ -50,7 +50,57 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const CENTURY_21 = '20';
+  const CENTURY_20 = '19';
+  const dateArr = date.split(`${fromFormat[3]}`);
```

Comment:
```suggestion
  const dateArr = date.split(fromFormat[3]);
```
You don't need interpolation in such cases.